v3/plugins/eclipse/cloud-shell/che-machine-exec-plugin/nightly/meta.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v2
```

Comment:
The directory structure you're using is incompatible with the `publisher/name/version` plugin IDs we use (name would be `cloud-shell/che-machine-exec-plugin`); perhaps just `eclipse/cloud-shell/nightly` is sufficient?

Reply:
Actually, to match the publisher/name/version in meta.yaml, this meta.yaml *has* to be at `v3/plugins/eclipse/cloud-shell/nigthly/meta.yaml` or CI will fail.

Reply:
Thanks, fixed.